machine_translation/prepare_data.py

```diff
@@ -160,7 +161,7 @@ def merge_parallel(src_filename, trg_filename, merged_filename):
                 while True:
                     lline = left.readline()
                     rline = right.readline()
-                    if (lline == '') or (rline == ''):
+                    if (lline == '') and (rline == ''):
```

Comment:
I finally understood what the code from this PR does. I think it can be done better:

```
# equizip can be found in picklable_itertools
# it ensures that your files have the same number of lines
for lline, rline in equizip(left, right):
    # do what you do now, but keep in mind that lline and rline have '\n' at the end
```